File: android/src/main/java/com/reactnativecloudstore/CloudStoreModule.java
package com.reactnativecloudstore;

import androidx.annotation.NonNull;

import com.facebook.react.bridge.Promise;
import com.facebook.react.bridge.ReactApplicationContext;
import com.facebook.react.bridge.ReactContextBaseJavaModule;
import com.facebook.react.bridge.ReactMethod;
import com.facebook.react.module.annotations.ReactModule;

@ReactModule(name = CloudStoreModule.NAME)
public class CloudStoreModule extends ReactContextBaseJavaModule {
  public static final String NAME = "CloudStore";

  public CloudStoreModule(ReactApplicationContext reactContext) {
    super(reactContext);
  }

  @Override
  @NonNull
  public String getName() {
    return NAME;
  }
}

File: android/src/main/java/com/reactnativecloudstore/CloudStoreModule.kt
package com.reactnativecloudstore

import com.facebook.react.bridge.ReactApplicationContext
import com.facebook.react.bridge.ReactContextBaseJavaModule
import com.facebook.react.bridge.ReactMethod
import com.facebook.react.bridge.Promise

class CloudStoreModule(reactContext: ReactApplicationContext) : ReactContextBaseJavaModule(reactContext) {

    override fun getName(): String {
        return "CloudStore"
    }

}
